### DISCUSSION TITLE
"Has Sync'ed" indication in the core registers

### DISCUSSION THREAD

**Discussion by Poofjunior** [harp-tech/protocol#89] [Q&A]
Hi folks,

Is there a way to tell when a Harp device has synchronized from an external signal?

I'm imagining some sort of core register bit that gets set if the device has synchronized to at least one external sync signal since boot. This is useful because lots of future code could easily start scheduling times based on the device's local time upon setup. If the system synchronizes at some unknown later time, it's possible in some implementations that the scheduled times will be invalid. Being able to check against whether or not the current time was altered would at least give us the ability to change times on currently scheduled tasks when the local time has changed.

**Comment by bruno-f-cruz** [comment]
Related to https://github.com/harp-tech/protocol/issues/8
